src/otx/core/model/anomaly.py

```diff
@@ -154,37 +155,6 @@ def configure_callbacks(self) -> list[Callback]:
             ),
         ]
 
-    def on_test_batch_end(
```

Comment:
It's just a question. Is it ok to remove? It seems that there is no corresponding code addition to this.

Reply:
Yeah. I think we can remove configure callbacks as well. I'll update the PR

Reply:
Yeah. I just tested it right now.